docs/tutorial/mac-app-store-submission-guide.md

```diff
@@ -1,29 +1,28 @@
 # Mac App Store Submission Guide
 
-Since v0.34.0, Electron allows submitting packaged apps to Mac App Store (MAS),
-this guide provides information on how to submit your app, and the limitations
+Since v0.34.0, Electron allows submitting packaged apps to the Mac App Store (MAS).
+This guide provides information on: how to submit your app & the limitations
 of the MAS build.
 
-## How to submit your app
+## How to Submit Your App
 
-Following steps introduces a simple way to submit your app to Mac App Store, but
-it doesn't make sure your app gets approved by Apple, you still have to read
-apple's [Submitting Your App][submitting-your-app] guide on how to meet Mac
-App Store's requirements.
+The following steps introduce a simple way to submit your app to Mac App Store. However, these steps
```

Comment:
This line and a couple others need to wrap because they go :arrow_right: of 80 cols.